### ISSUE TITLE
"Under development"

### ISSUE THREAD

**Issue by AboHadi313** [NikolasEnt/Traffic-Sign-Classifier#2]
Hello brother,
I'm interested on the second CNN, which is the speed limits recognition.
I used a pre-trained model, and tried to apply the train.py thing on it, although the loss was slightly under 1, I couldn't detect any picture correctly !
So, I just wanna make sure, can you use the dataset "German" to retrain a pre-trained model ? Or is it just a waste of time ? And I should create my own CNN ? I'm still a student and new to Tensorflow so it is hard for me to do everything from scratch. :(
Any new progress/success with your project ?

**Comment by NikolasEnt** [comment]
Hi!
Thank you for interest in my project.
What do yo mean by "German"? I have only the dataset, provided by Udacity.

Keep in mind, that the project was implemented quit long ago, so, modern versions of libraries may be not compatible with it.
BTW, there is no train.py
